test/screens_web/views/v2/audio/elevator_status_view_test.exs

```diff
@@ -0,0 +1,79 @@
+defmodule ScreensWeb.V2.Audio.ElevatorStatusViewTest do
+  use ScreensWeb.ConnCase, async: true
+  alias ScreensWeb.V2.Audio.ElevatorStatusView
+
+  describe "No elevator alerts" do
+    test "renders all clear message" do
+      assigns = %{
+        active_at_home_pages: [],
+        list_pages: [],
+        upcoming_at_home_pages: [],
+        elsewhere_pages: []
+      }
+
+      assert render(assigns) ==
+               "\n    <p>Elevator Closures across the T.</p>\n    <p>All elevators are working at this station.</p>\n    <p>Other elevator closures:</p>\n    <p>All other MBTA elevators are working or have a backup elevator within 20 feet.</p>\n    "
+    end
+  end
+
+  describe "Elevator alert" do
+    test "with no additional closures renders closure message + other elevators working message" do
+      assigns = %{
+        active_at_home_pages: test_stations(),
+        list_pages: [],
+        upcoming_at_home_pages: [],
+        elsewhere_pages: []
+      }
+
+      assert render(assigns) =~ "take a different elevator :)"
+
+      assert render(assigns) =~
+               "All other MBTA elevators are working or have a backup elevator within 20 feet."
+    end
+
+    test "with additional closures renders closure message + additional closures message" do
+      assigns = %{
+        active_at_home_pages: test_stations(),
+        list_pages: test_stations(),
+        upcoming_at_home_pages: [],
+        elsewhere_pages: []
+      }
+
+      assert render(assigns) =~ "take a different elevator :)"
+      assert render(assigns) =~ "For a full list of elevator alerts"
```

Comment:
For this test, it would be nice to have at least two different test stations that are being checked for in the `render(assigns)` instead of the same elevator alert twice. But I see that the logic is working now so it's not strictly necessary